### PR TITLE
Add a more welcoming welcome message, aims of guide and spaces

### LINKED ISSUE #163: Add welcome message
Add a welcome message to the reader and the aims of the UG

### PR DESCRIPTION
**Pull request by josiahkhoo**
- Add a more welcoming welcome message (Closes #163)
- Add aims of UG
- Fix spacing and warning divs on Test, Stats, Import and Export Command.

Please give feedback on the welcome message cause I don't know if its welcoming enough or if certain sections are redundant @joshtyf

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,11 +3,13 @@
...
 
-# Your guide to QuickCache
+# Welcome to QuickCache User Guide
 
 **QuickCache** is a desktop app for managing flashcards, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). 
 
-If you are a student who can type fast and loves organizing your study materials, QuickCache can get your flashcard management tasks done faster than any traditional GUI appplication.
+If you are a student who can type fast and loves organizing your study materials, **QuickCache** can get your flashcard management tasks done faster than any traditional GUI appplication.
+
+The **QuickCache User Guide** helps you get started with using **QuickCache**. This user guide aims to walk you through the robust features that **QuickCache** has to offer and take you from zero to hero. Don't like reading? A summary is available at the end for your reference.
```
Comment (ChenXJ98) [inline docs/UserGuide.md:13]: Very welcoming! Thanks!!

**Review by ChenXJ98 — APPROVED** [review]

File: docs/UserGuide.md [hunk 2]
```diff
@@ -3,11 +3,13 @@
...
```
Comment (joshtyf) [inline docs/UserGuide.md:12]: Nice. I think can consider adding a link to the words "summary" that will link to the command summary section

File: docs/UserGuide.md [hunk 3]
```diff
@@ -330,40 +334,54 @@ Format: `clear`
...
+
     ![TestStep1](./images/TestStep1.png)
+    
 2. Using the indices of the displayed list, enter the `test` command followed by the index of the flashcard you want to test and what you think the answer to the question is. For example, if you want to test the second flashcard in the displayed list with the answer `a computer organization module`, you can enter `test 2 ans/a computer organization module`.
+
+    <div markdown="block" class="alert alert-info">
+    :bulb: Answer is <b>case insensitive</b>.
+    </div>
+    <div markdown="block" class="alert alert-danger">
+    The index <b>must be a positive integer</b> 1, 2, 3, ...
```
Comment (joshtyf) [inline docs/UserGuide.md:349]: I think can add an `:exclamation:`❗ emoji for this alert class. Same for the rest of the `alert-danger` classes

**Review by joshtyf — COMMENTED** [review]
Looks good. I suggested some really minor changes. Thanks for helping!

**Pull request merged**